css/quiz.css

```diff
@@ -0,0 +1,77 @@
+@import url(https://fonts.googleapis.com/css?family=Work+Sans:300,600);
+
+
+body{
+	font-size: 20px;
+  background-color: #000;
+	font-family: 'Work Sans', sans-serif;
```

Comment:
Instead of using a new font again use the font that is already being used in home page.